CDTDatastore/query/CDTQIndexManager.h

```diff
@@ -75,47 +77,49 @@ typedef NS_ENUM(NSInteger, CDTQQueryError) {
 /**
  Constructs a new CDTQIndexManager which indexes documents in `datastore`
  */
-+ (CDTQIndexManager *)managerUsingDatastore:(CDTDatastore *)datastore
-                                      error:(NSError *__autoreleasing *)error;
++ (nullable CDTQIndexManager *)
+managerUsingDatastore:(CDTDatastore *)datastore
+                error:(NSError *__autoreleasing __nullable *__nullable)error;
 
-- (instancetype)initUsingDatastore:(CDTDatastore *)datastore
-                             error:(NSError *__autoreleasing *)error;
+- (nullable instancetype)initUsingDatastore:(CDTDatastore *)datastore
+                                      error:(NSError *__autoreleasing __nullable *__nullable)error;
 
-- (NSDictionary * /* NSString -> NSArray[NSString]*/)listIndexes;
+- (NSDictionary<NSString *, NSArray<NSString *> *> *)listIndexes;
 
 /** Internal */
-+ (NSDictionary /* NSString -> NSArray[NSString]*/ *)listIndexesInDatabaseQueue:
-        (FMDatabaseQueue *)db;
++ (NSDictionary<NSString *, NSArray<NSString *> *> *)listIndexesInDatabaseQueue:
+    (FMDatabaseQueue *)db;
 /** Internal */
-+ (NSDictionary /* NSString -> NSArray[NSString]*/ *)listIndexesInDatabase:(FMDatabase *)db;
++ (NSDictionary<NSString *, NSArray<NSString *> *> *)listIndexesInDatabase:(FMDatabase *)db;
 
-- (NSString *)ensureIndexed:(NSArray * /* NSString */)fieldNames withName:(NSString *)indexName;
+- (NSString *)ensureIndexed:(NSArray<NSString *> *)fieldNames withName:(NSString *)indexName;
```

Comment:
I think this method can return `nil`. For example, if `indexName` is of zero length.